### ISSUE TITLE
[How] can we continuous comment next line?

### ISSUE THREAD

**Issue by 2nthony** [LunarVim/LunarVim#3237]
## expected

from my old neovim

![auto-comment](https://user-images.githubusercontent.com/19513289/195881452-5947bbeb-731f-4c21-9b9f-c14859af15ba.gif)

## actually

in lunarvim

![no-auto-comment](https://user-images.githubusercontent.com/19513289/195881648-5795c13e-ece5-40f2-b9e8-8db92f2f4dac.gif)

### Support information

I am not sure is there has a plugin that can implement this feature.

**Comment by abzcoding** [comment]
add this to your `~/.config/lvim/config.lua`
```lua
vim.api.nvim_clear_autocmds { pattern = "*", group = "_format_options" }
```

this happens because we have this:
https://github.com/LunarVim/LunarVim/blob/1fbdcabf1915b35599c7912a45ef92888417ea65/lua/lvim/core/autocmds.lua#L87-L94


@ChristianChiarulli we should probably don't force this on users

@kylo252 what do you think?

**Comment by ChristianChiarulli** [comment]
I always found it pretty annoying tbh. I don't mind removing it tho.

**Comment by opalmay** [comment]
I think it would be nice to have backspace on the new line toggle the comment off (rather than having to go into normal mode or pressing backspace a bunch of times). Not sure how this would be implemented, though.

**Comment by 2nthony** [comment]
> For my opinion this options must be personal. For me this is bad variant because only I`m know what must be on the next line - coment/space/code VIM way - is backward that you proposition. You write the text - any in edit mode. Next you going to visual mode and do what are you need with your text. For example - select lines by Shift + v and j k and press SPS + / to comment selected block.

sounds correct, this is actually personal.

the point is: if other users want this, they can easy to find this solution and add it to their config.

**Comment by Aniketmishra0** [comment]
Hello Assign me to work on this

**Comment by abzcoding** [comment]
fixed via #3278
